frontend/src/views/web/featured-challenge/leaderboard.html

```diff
@@ -41,18 +41,50 @@
                                 <table ng-if="featured_challenge.isResult && featured_challenge.leaderboard.length" class="highlight">
                                     <thead>
                                         <tr>
-                                            <td class="align-left" data-field="rank">Rank</td>
-                                            <td class="align-left" data-field="team">Participant Team</td>
-                                            <td class="align-right" ng-repeat="key in featured_challenge.leaderboard[0].leaderboard__schema.labels">{{key}}</td>
-                                            <td class="align-right" data-field="submission_time">Last Submission at</td>
+                                            <td class="align-left" data-field="rank">
+                                                <a href="#" ng-click="featured_challenge.sortFeaturedLeaderboard(featured_challenge, 'rank');">
+                                                    Rank
+                                                    <span class="fa-stack fa-1x">
+                                                            <i class="fa fa-sort-asc fa-stack-1x" ng-class="featured_challenge.reverseSort && featured_challenge.sortColumn == 'rank'? 'text-dark-black' : 'text-light-black w-500'"></i>
+                                                            <i class="fa fa-sort-desc fa-stack-1x" ng-class="!featured_challenge.reverseSort && featured_challenge.sortColumn == 'rank'? 'text-dark-black' : 'text-light-black w-500'"></i>
+                                                    </span>
+                                                </a>
+                                            </td>
+                                            <td class="align-left" data-field="team">
+                                                <a href="#" ng-click="featured_challenge.sortFeaturedLeaderboard(featured_challenge, 'string');">
+                                                    Participant Team
```

Comment:
Have this header wrapped in a similar span tag that of the regular challenge page.
Any specific reason why you didn't have then in a span tag with font-size and font-weight?                                                 Example: 

```suggestion
                                                    <span class="w-400 fs-18">Participant Team</span>
```